src/hooks/useTimestamps.js

```diff
@@ -41,6 +38,7 @@ function useTimestamps(defaultTimestamps) {
     addNewTimestamp,
     deleteTimestamp,
     updateTimestamp,
+    resetTimestamp,
```

Comment:
I think I would like to gather `actions` as an object field more, so do `context`, `state`,
maybe it will be clearer,
```javascript
{
  state,
  actions: { ... },
  context: state.context,
  // ...
}
```

Reply:
Thank you for your response. I update the `actions` part, but stick to use `matches: state.matches` since I think I don't need to pass all the value in `state`. However, maybe `context`, `matches`, and `value` are basically all the needed value?